exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -58,6 +89,54 @@ static HashCode parseSubmitTxResponse(String json) {
     HashCode hash;
   }
 
+  /**
+   * Json object wrapper for get transaction response.
+   */
+  @Value
+  private static class GetTxResponse {
+    @NonNull
+    TransactionStatus type;
+    @NonNull
+    GetTxResponseContent content;
+    TransactionLocation location;
+    @SerializedName("location_proof")
+    JsonObject locationProof; //TODO: in scope of LC P3
+    GetTxResponseExecutionResult status;
+  }
+
+  /**
+   * Json object wrapper for get transaction response content i.e.
+   * {@code "$.content"}.
+   */
+  @Value
+  private static class GetTxResponseContent {
+    JsonObject debug; // contains executable tx in json. currently not supported
+    @NonNull
+    TransactionMessage message;
+  }
+
+  /**
+   * Json object wrapper for transaction execution result i.e.
+   * {@code "$.status"}.
+   */
+  @Value
+  private static class GetTxResponseExecutionResult {
+    GetTxResponseExecutionStatus type;
+    int code;
+    String description;
+  }
+
+  /**
+   * Json object wrapper for transaction execution status i.e.
+   * {@code "$.status.type"}.
+   */
+  private enum GetTxResponseExecutionStatus {
+    @SerializedName("success")
+    SUCCESS,
+    @SerializedName("error")
+    ERROR
```

Comment:
Would you double-check if there is "panic" type?

Reply:
good catch. there is panic as well